From e11a1ff002fdc7aab81f40b99fcea2fc86eba85b Mon Sep 17 00:00:00 2001
From: miao_kaibo <miaokaibo@outlook.com>
Date: Fri, 2 Apr 2021 10:23:47 +0800
Subject: [PATCH] rename source iso

---
 isomaker/init.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/isomaker/init.sh b/isomaker/init.sh
index f708555..2603bfb 100755
--- a/isomaker/init.sh
+++ b/isomaker/init.sh
@@ -112,7 +112,7 @@ function global_var_init()
         DBG_ISO_NAME="${PRODUCT}-${VERSION}-${RELEASE}-debug-${ARCH}-dvd.iso"
         EVE_ISO_NAME="${PRODUCT}-${VERSION}-${RELEASE}-everything-${ARCH}-dvd.iso"
         EVE_DEBUG_ISO_NAME="${PRODUCT}-${VERSION}-${RELEASE}-everything-debug-${ARCH}-dvd.iso"
-        EVE_SRC_ISO_NAME="${PRODUCT}-${VERSION}-${RELEASE}-everything-source-dvd.iso"
+        EVE_SRC_ISO_NAME="${PRODUCT}-${VERSION}-${RELEASE}-source-dvd.iso"
         NETINST_ISO_NAME="${PRODUCT}-${VERSION}-${RELEASE}-netinst-${ARCH}-dvd.iso"
     else
         RELEASE_NAME="${PRODUCT}-${VERSION}-${ARCH}"
@@ -121,7 +121,7 @@ function global_var_init()
         DBG_ISO_NAME="${PRODUCT}-${VERSION}-debug-${ARCH}-dvd.iso"
         EVE_ISO_NAME="${PRODUCT}-${VERSION}-everything-${ARCH}-dvd.iso"
         EVE_DEBUG_ISO_NAME="${PRODUCT}-${VERSION}-everything-debug-${ARCH}-dvd.iso"
-        EVE_SRC_ISO_NAME="${PRODUCT}-${VERSION}-everything-source-dvd.iso"
+        EVE_SRC_ISO_NAME="${PRODUCT}-${VERSION}-source-dvd.iso"
         NETINST_ISO_NAME="${PRODUCT}-${VERSION}-netinst-${ARCH}-dvd.iso"
     fi
 
-- 
2.30.0

