From 78879f08b31ea910e7b39b512a74986788645769 Mon Sep 17 00:00:00 2001
From: miao_kaibo <miaokaibo@outlook.com>
Date: Sat, 13 Mar 2021 10:11:53 +0800
Subject: [PATCH] bug fix I3B7CH

---
 rpm.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rpm.sh b/rpm.sh
index 7e4a7e2..9a255a7 100755
--- a/rpm.sh
+++ b/rpm.sh
@@ -148,7 +148,7 @@ function get_rpm_pub_key()
 
 function get_everything_rpms()
 {
-    yum list --installroot="${BUILD}"/tmp --available | awk '{print $1}' | grep -E "noarch|${ARCH}" | grep -v "debuginfo" | grep -v "debugsource" > ava_every_lst
+    yum list --installroot="${BUILD}"/tmp --available | awk '{print $1}' | grep -E "\.noarch|\.${ARCH}" | grep -v "debuginfo" | grep -v "debugsource" > ava_every_lst
     parse_rpmlist_xml "exclude"
     cat parsed_rpmlist_exclude
     if [ -s parsed_rpmlist_exclude ];then
-- 
2.27.0

