From d9c43d5200adf079a1a5b3705048a740cc8cffc8 Mon Sep 17 00:00:00 2001
From: Wenlong Zhang <zhangwenlong@loongson.cn>
Date: Mon, 3 Apr 2023 12:43:07 +0800
Subject: [PATCH] add loongarch64 support for runtime install

Signed-off-by: Wenlong Zhang <zhangwenlong@loongson.cn>
---
 isomaker/80-openeuler/runtime-install.tmpl | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/isomaker/80-openeuler/runtime-install.tmpl b/isomaker/80-openeuler/runtime-install.tmpl
index 1856aff..035d6c3 100644
--- a/isomaker/80-openeuler/runtime-install.tmpl
+++ b/isomaker/80-openeuler/runtime-install.tmpl
@@ -47,6 +47,11 @@ installpkg glibc-all-langpacks
     installpkg syslinux
     installpkg grub2-tools grub2-tools-minimal grub2-tools-extra
 %endif
+%if basearch == "loongarch64":
+    installpkg efibootmgr
+    installpkg grub2-efi-loongarch64 grub2-efi-loongarch64-cdboot grub2-efi-loongarch64-modules 
+    installpkg grub2-tools grub2-tools-minimal grub2-tools-extra
+%endif
 
 ## yay, plymouth
 installpkg plymouth
@@ -112,7 +117,7 @@ installpkg pciutils usbutils ipmitool
 installpkg smartmontools
 installpkg hdparm
 ## see bug #1483278
-%if basearch not in ("arm", "armhfp"):
+%if basearch not in ("arm", "armhfp", "loongarch64"):
     installpkg libmlx4 rdma-core
 %endif
 installpkg rng-tools
-- 
2.33.0

