From bb1688a8e8c8dc18ccd3c2ed778a0c5c0bd52633 Mon Sep 17 00:00:00 2001
From: wangchong1995924 <15229716099@163.com>
Date: Wed, 26 Jan 2022 14:13:00 +0800
Subject: [PATCH 1/1] support usb flash drive mode

---
 isomaker/iso.sh | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/isomaker/iso.sh b/isomaker/iso.sh
index b15b394..379c538 100755
--- a/isomaker/iso.sh
+++ b/isomaker/iso.sh
@@ -39,6 +39,7 @@ function gen_standard_iso()
     if [ "$ARCH" == "x86_64" ]; then
         mkisofs -R -J -T -r -l -d -joliet-long -allow-multidot -allow-leading-dots -no-bak -V "${RELEASE_NAME}" -o "${OUTPUT_DIR}/${STANDARD_ISO_NAME}" -b isolinux/isolinux.bin -c isolinux/boot.cat -no-emul-boot -boot-load-size 4 -boot-info-table  -eltorito-alt-boot -e images/efiboot.img -no-emul-boot "${BUILD}"/iso
         [ $? != 0 ] && return 1
+	isohybrid -u "${OUTPUT_DIR}/${STANDARD_ISO_NAME}"
     elif [ "$ARCH" == "aarch64" ]; then
         mkisofs -R -J -T -r -l -d -joliet-long -allow-multidot -allow-leading-dots -no-bak -V "${RELEASE_NAME}" -o "${OUTPUT_DIR}/${STANDARD_ISO_NAME}" -e images/efiboot.img -no-emul-boot "${BUILD}"/iso
         [ $? != 0 ] && return 1
@@ -75,6 +76,7 @@ function gen_everything_iso()
     if [ "$ARCH" == "x86_64"  ]; then
         mkisofs -R -J -T -r -l -d -joliet-long -allow-multidot -allow-leading-dots -no-bak -V "${RELEASE_NAME}" -o /result/"${EVE_ISO_NAME}" -b isolinux/isolinux.bin -c isolinux/boot.cat -no-emul-boot -boot-load-size 4 -boot-info-table  -eltorito-alt-boot -e images/efiboot.img -no-emul-boot "${BUILD}"/iso
         [ $? != 0  ] && return 1
+	isohybrid -u /result/"${EVE_ISO_NAME}"
     elif [ "$ARCH" == "aarch64"  ]; then
         mkisofs -R -J -T -r -l -d -joliet-long -allow-multidot -allow-leading-dots -no-bak -V "${RELEASE_NAME}" -o /result/"${EVE_ISO_NAME}" -e images/efiboot.img -no-emul-boot "${BUILD}"/iso
         [ $? != 0  ] && return 1
@@ -126,6 +128,7 @@ function gen_netinst_iso()
     if [ "$ARCH" == "x86_64"  ]; then
         mkisofs -R -J -T -r -l -d -joliet-long -allow-multidot -allow-leading-dots -no-bak -V "${RELEASE_NAME}" -o /result/"${NETINST_ISO_NAME}" -b isolinux/isolinux.bin -c isolinux/boot.cat -no-emul-boot -boot-load-size 4 -boot-info-table  -eltorito-alt-boot -e images/efiboot.img -no-emul-boot "${BUILD}"/iso
         [ $? != 0  ] && return 1
+	isohybrid -u /result/"${NETINST_ISO_NAME}"
     elif [ "$ARCH" == "aarch64"  ]; then
         mkisofs -R -J -T -r -l -d -joliet-long -allow-multidot -allow-leading-dots -no-bak -V "${RELEASE_NAME}" -o /result/"${NETINST_ISO_NAME}" -e images/efiboot.img -no-emul-boot "${BUILD}"/iso
     fi
-- 
2.27.0

