From 6958c40cac406dc5636189b9cbd8635ebdbe72b3 Mon Sep 17 00:00:00 2001
From: xyn-coder <xiangyuning@huawei.com>
Date: Tue, 22 Nov 2022 16:16:38 +0800
Subject: [PATCH] add parse_everything_deb_exclude

---
 isomaker/rpm.sh | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/isomaker/rpm.sh b/isomaker/rpm.sh
index 85b1042..0f77b34 100755
--- a/isomaker/rpm.sh
+++ b/isomaker/rpm.sh
@@ -242,6 +242,14 @@ function everything_debug_rpms_download()
 {
     mkdir ${EVERY_DEBUG_DIR}
     yum list --installroot="${BUILD}"/tmp --available | awk '{print $1}' | grep -E "debuginfo|debugsource" > ava_debug_lst
+    parse_rpmlist_xml "everything_debug_exclude"
+    cat parsed_rpmlist_everything_debug_exclude
+    if [ -s parsed_rpmlist_everything_debug_exclude ];then
+        for rpmname in $(cat parsed_rpmlist_everything_debug_exclude)
+        do
+            sed -i "/^${rpmname}\./d" ava_debug_lst
+        done
+    fi
     yumdownloader --resolve --installroot="${BUILD}"/tmp --destdir="${EVERY_DEBUG_DIR}" $(cat ava_debug_lst | tr '\n' ' ')
     if [ $? != 0 ] || [ $(ls ${EVERY_DEBUG_DIR} | wc -l) == 0 ]; then
         echo "yumdownloader with --resolve failed, trying to yumdownloader without --resolve"
-- 
2.33.0

