From efcc2fd150f9fe625a03b4b5d9c492f6691b1ba1 Mon Sep 17 00:00:00 2001
From: small_leek <xiasenlin1@huawei.com>
Date: Fri, 18 Mar 2022 11:19:07 +0800
Subject: [PATCH] add parse_everything_deb_exclude

---
 isomaker/rpm.sh | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/isomaker/rpm.sh b/isomaker/rpm.sh
index e5fad1f..fa6167b 100644
--- a/isomaker/rpm.sh
+++ b/isomaker/rpm.sh
@@ -217,6 +217,14 @@ function everything_debug_rpms_download()
 {
     mkdir ${EVERY_DEBUG_DIR}
     yum list --installroot="${BUILD}"/tmp --available | awk '{print $1}' | grep -E "debuginfo|debugsource" > ava_debug_lst
+    parse_rpmlist_xml "everything_debug_exclude"
+    cat parsed_rpmlist_everything_debug_exclude
+    if [ -s parsed_rpmlist_everything_debug_exclude ];then
+        for rpmname in $(cat parsed_rpmlist_everything_debug_exclude)
+        do
+            sed -i "/^${rpmname}\./d" ava_debug_lst
+        done
+    fi
     yumdownloader --resolve --installroot="${BUILD}"/tmp --destdir="${EVERY_DEBUG_DIR}" $(cat ava_debug_lst | tr '\n' ' ')
     if [ $? != 0 ] || [ $(ls ${EVERY_DEBUG_DIR} | wc -l) == 0 ]; then
         echo "yumdownloader with --resolve failed, trying to yumdownloader without --resolve"
-- 
2.27.0

