From 90c2319b0fbde5e263f999288627cc67ec5fc9c0 Mon Sep 17 00:00:00 2001
From: wangchong1995924 <15229716099@163.com>
Date: Thu, 14 Jul 2022 17:44:21 +0800
Subject: Don't clean up libdiff and libcairo-script*

---
 isomaker/80-openeuler/runtime-cleanup.tmpl | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/isomaker/80-openeuler/runtime-cleanup.tmpl b/isomaker/80-openeuler/runtime-cleanup.tmpl
index 35b2c83..5bce798 100644
--- a/isomaker/80-openeuler/runtime-cleanup.tmpl
+++ b/isomaker/80-openeuler/runtime-cleanup.tmpl
@@ -67,7 +67,7 @@ removefrom iptables --allbut \
    
 removepkg libXxf86misc
 removepkg libasyncns libhbaapi libhbalinux
-removepkg libmcpp libtiff linux-atm-libs
+removepkg libmcpp linux-atm-libs
 removepkg lvm2-libs m4 mailx makebootfat mcpp
 removepkg mingetty mobile-broadband-provider-info pkgconfig ppp pth
 removepkg rmt rpcbind squashfs-tools system-config-firewall-base
@@ -146,7 +146,7 @@ removefrom bind-utils /usr/bin/dig /usr/bin/host /usr/bin/nsupdate
 removefrom bitmap-fangsongti-fonts /usr/share/fonts/*
 removefrom ca-certificates /etc/pki/java/*
 removefrom ca-certificates /etc/pki/tls/certs/ca-bundle.trust.crt /etc/ssl/*
-removefrom cairo /usr/${libdir}/libcairo-script* /usr/bin/cairo-sphinx
+removefrom cairo /usr/bin/cairo-sphinx
 removefrom coreutils /etc/* /usr/bin/link /usr/bin/nice /usr/bin/stty /usr/bin/su /usr/bin/unlink
 removefrom coreutils /usr/sbin/runuser /usr/bin/[ /usr/bin/base64 /usr/bin/chcon
 removefrom coreutils /usr/bin/cksum /usr/bin/comm /usr/bin/csplit
-- 
2.27.0

