From 6d04f0dd0695b0408ed499fb98213eb6ba52139b Mon Sep 17 00:00:00 2001
From: wangchong1995924 <15229716099@163.com>
Date: Wed, 19 Apr 2023 15:18:17 +0800
Subject: [PATCH] fix make edge iso error


diff --git a/isomaker/80-openeuler/runtime-install.tmpl b/isomaker/80-openeuler/runtime-install.tmpl
index 1856aff..a6144bf 100644
--- a/isomaker/80-openeuler/runtime-install.tmpl
+++ b/isomaker/80-openeuler/runtime-install.tmpl
@@ -22,7 +22,7 @@ installpkg --optional *-firmware --except alsa* --except midisport-firmware \
                        --except crystalhd-firmware --except ivtv-firmware \
                        --except cx18-firmware --except iscan-firmware \
                        --except uhd-firmware --except lulzbot-marlin-firmware \
-                       --except arm-trusted-firmware
+                       --except arm-trusted-firmware --except raspberrypi-firmware
 installpkg b43-openfwwf
 
 ## install all of the glibc langpacks since otherwise we get no locales
-- 
2.27.0

